ArrayOps.java

```diff
@@ -1,26 +1,173 @@
 public class ArrayOps {
     public static void main(String[] args) {
-        
+        //int[] array={1,-2,3,-4,5};
+        //System.out.println(findMissingInt(array));
+        //System.out.println(secondMaxValue(array));
+        int [] array2={1,2,3,-4,5};
+        int [] array1={1,3,-4,5};
+        System.out.println(containsTheSameElements(array1,array2));
+        //int [] array={1,1,1,2,2};
+        //System.out.println(isSorted(array));
     }
     
+    //returns the missing integer in the array, between 0 and the array length
     public static int findMissingInt (int [] array) {
         // Write your code here:
-        return -1;
+        int sumArray=0;
+        int supposedSumArray=0;
+        int missing=0;
+
+
+        for(int i=1;i<=array.length;i++)
+        {
+            supposedSumArray+=i;
+        }
+
+        for(int j=0;j<array.length;j++)
+        {
+            sumArray+=array[j];
+        }
```

Comment:
You could combine both loops - to do that you should initialize your variables differently.
```suggestion
        for(int I=0;i<array.length;i++)
        {
            supposedSumArray+=i;
            sumArray+=array[j];
        }
```